visualization/test/text_simple_multiport.cpp

```diff
@@ -0,0 +1,30 @@
+#include <pcl/point_types.h>
+
+#include <pcl/visualization/pcl_visualizer.h>
+
+int
+main (int argc, char** argv)
+{
+  pcl::visualization::PCLVisualizer viz ("Visualizator");
+  int leftPort(0);
+  int rightPort(0);
+
+  viz.createViewPort(0, 0, 0.5, 1, leftPort);
+  viz.createViewPort(0.5, 0, 1, 1, rightPort);
+
+  viz.addCoordinateSystem(1.0);
```

Comment:
Space before the parenthesis from line 9 to 15.

Reply:
Ok, it's done.